### ISSUE TITLE
Add build instruction to the README

### ISSUE THREAD

**Issue by approximator** [approximator/waied#3]
- How to clone with submodules
- Which dependencies have to be installed
- build commands

**Comment by SergeyKarasyov** [comment]
Hi Mr. Approximator, 
I've built the project on on Mint v19
$uname -a
Linux computer 4.15.0-20-generic #21-Ubuntu SMP Tue Apr 24 06:16:15 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux

What was installed:
1) Set of applications/libraries required for build
_apt install software-properties-common git cmake libssl-dev libcurl4-openssl-dev g++ mesa-common-dev libglu1-mesa-dev_

2) Installation from this step may be omitted, as it didn't help. Unfortunately, the qt5.9.5(which is available in mint repos) doesn't have some libs required for application to run(exactly -> build was successful, but the application didn't run). 
_sudo add-apt-repository ppa:beineri/opt-qt-5.10.0-xenial
apt update
sudo apt-get install qtdeclarative5-dev libvlccore-dev libvlc-dev qtcreator qt5
sudo apt-get install qt5-default 
sudo apt-get install qtmultimedia5-dev 
sudo apt-get install qtquickcontrols2-5-dev_


3) Hence I used installation with an installer -> 
_wget http://download.qt.io/official_releases/qt/5.11/5.11.2/qt-opensource-linux-x64-5.11.2.run
chmod +x qt-opensource-linux-x64-5.11.2.run
./qt-opensource-linux-x64-5.11.2.run_
